packages/account/src/Components/forms/form-select-field.tsx

```diff
@@ -31,7 +31,7 @@ const FormSelectField: React.FC<TFormSelectField> = ({
     const onSelect =
         (field: string, setFieldValue: TSetFieldValue) =>
         ({ value, text }: TListItem) => {
-            setFieldValue(field, value ? text : '', true);
+            setFieldValue(field, (value && text) ?? '', true);
```

Comment:
as per this logic,
if (value && text) is true, set field value to `true` else if (value && text) is null or undefined, set field value to `''`, right?
just checking if this is correct/intended.

Reply:
<img width="215" alt="Screenshot 2023-11-22 at 3 15 36 PM" src="https://github.com/binary-com/deriv-app/assets/103026762/953966cc-4516-40e2-95fb-ee0560930791">

It will return `text` instead of true in this logic. The screenshot attached is the example output of the logic 🙏